loops-and-arrays.js

```diff
@@ -12,15 +12,36 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
+console.log(names);
 
+for (let i = 0; i < names.length - 1; i++) {
```

Comment:
Hier heb je na de names.length een -1 staan, dit zorgt ervoor dat "Joop" niet mee gaat in je loop.